ocs_ci/ocs/platform_nodes.py

```diff
@@ -410,6 +417,22 @@ def create_and_attach_nodes_to_cluster(self, node_conf, node_type, num_nodes):
         node_cls_obj = node_cls(node_conf, node_type, num_nodes)
         node_cls_obj.add_node()
 
+    def terminate_nodes(self, nodes, wait=True):
+        """
+        Terminate the VMs
+
+        Args:
+            nodes (list): The OCS objects of the nodes
+            wait (bool): True for waiting the VMs to terminate,
+            False otherwise
+
+        """
+        vms = self.get_vms(nodes)
+        self.vsphere.remove_vms_from_inventory(vms)
+        if wait:
+            for vm in vms:
+                self.vsphere.wait_for_vm_delete(vm)
+
```

Comment:
move function definition out of base class ( in this case VMWareNodes ). reason is if someone call terminate_nodes from inherited class ( UPI ), it gives unexpected results.

Reply:
Okay, I will do that.

Reply:
Yes, this needs to be done

Reply:
done here https://github.com/red-hat-storage/ocs-ci/pull/5109/files#diff-478e5b871f475f1e312384c0eeb216726f11da23e3d3ac4df80aa3ee5d92e441R186